src/index.html

```diff
@@ -7,6 +7,410 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+
+          src="./images/logo.png"
+          alt="logo"
+          class="logo__image">
+
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a  href="#" class="nav__link is-active"> Apple </a>
+          </li>
+          <li class="nav__item">
+            <a  href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a  href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+
+             href="#"
+             class="nav__link"
+             data-qa="nav-hover"
+             >
+             Laptops & Computers
+
+            </a>
```

Comment:
```suggestion
            <a
             href="#"
             class="nav__link"
             data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
```

Reply:
don't need add spaces:
```suggestion
            <a
             href="#"
             class="nav__link"
             data-qa="nav-hover"
             >
             Laptops & Computers
            </a>
```